Replace CNRP entries with entirely new ones

Infernio opened the pull request:
Under #24 and #317.

Infernio committed:
Replace CNRP Bruma entries with new ones

Appropriate Invent tag already present in the plugin description.
Graphics because it changes the BMDT flags for a couple amulets.
All other changes are forwards from OOO/UOP.

Infernio committed:
Replace CNRP Chorrol entries with new ones

Same story as last time. I think I'm noticing a pattern :P

Infernio committed:
Replace CNRP DB entries with new ones

This time around even the Graphics tag is present in the description.

Infernio committed:
Replace CNRP Fighters Guild entries with new ones

Same story as CNRP DB, Graphics tag is already present.

Infernio committed:
Replace CNRP Cheydinhal entries with new ones

This time around we need to add the Graphics again.

sibir-ine commented on masterlist.yaml:
```diff
@@ -9836,78 +9836,50 @@ plugins:
-            text: 'Use ''CNRP Bruma4OOO.esp'' instead.'
-          - lang: de
-            text: 'Nutzen Sie stattdessen ''CNRP Bruma4OOO.esp''.'
+      - <<: *useInstead
+        subs: [ 'CNRP Bruma4OOO.esp' ]
```
`*useVersion` is a bit more appropriate in situations like these, but `*useInstead` would still work. Up to you.

```suggestion
      - <<: *useVersion
        subs: [ 'OOO/FCOM' ]
```

sibir-ine commented on masterlist.yaml:
```diff
@@ -9836,78 +9836,50 @@ plugins:
-            text: 'Use ''CNRP Chorrol4OOO.esp'' instead.'
-          - lang: de
-            text: 'Nutzen Sie stattdessen ''CNRP Chorrol4OOO.esp''.'
+      - <<: *useInstead
+        subs: [ 'CNRP Chorrol4OOO.esp' ]
```
Same situation as above.

```suggestion
      - <<: *useVersion
        subs: [ 'OOO/FCOM' ]
```

sibir-ine commented on masterlist.yaml:
```diff
@@ -9836,78 +9836,50 @@ plugins:
-            text: 'Use ''CNPR DB for OOO.esp'' instead.'
-          - lang: de
-            text: 'Nutzen Sie stattdessen ''CNPR DB for OOO.esp''.'
+      - <<: *useInstead
+        subs: [ '[CNPR DB for OOO.esp](https://www.nexusmods.com/oblivion/mods/29428)' ]
```
Same situation as above.

```suggestion
      - <<: *useVersion
        subs: [ '[OOO/FCOM](https://www.nexusmods.com/oblivion/mods/29428)' ]
```

sibir-ine commented on masterlist.yaml:
```diff
@@ -9836,78 +9836,50 @@ plugins:
-    tag: [ Invent ]
-  - name: 'CNRP DB.esp'
+
+  - name: 'CNRP DB( for OOO)?\.esp'
     url: [ 'https://www.nexusmods.com/oblivion/mods/28027' ]
+  - name: 'CNRP DB.esp'
```
Since the OOO patch is on a different page, it wouldn't make sense to assign it that location.

```suggestion
  - name: 'CNRP DB.esp'
    url: [ 'https://www.nexusmods.com/oblivion/mods/28027' ]
```

sibir-ine commented on masterlist.yaml:
```diff
@@ -9836,78 +9836,50 @@ plugins:
-            text: 'Use ''CNRP Cheydinhall4OOO.esp'' instead.'
-          - lang: de
-            text: 'Nutzen Sie stattdessen ''CNRP Cheydinhall4OOO.esp''.'
+      - <<: *useInstead
+        subs: [ 'CNRP Cheydinhal4OOO.esp' ]
```
Same situation as above.

```suggestion
      - <<: *useVersion
        subs: [ 'OOO/FCOM' ]
```

sibir-ine commented on masterlist.yaml:
```diff
@@ -9836,78 +9836,50 @@ plugins:
-            text: 'Use ''CNRP FightersGuild4OOO.esp'' instead.'
-          - lang: de
-            text: 'Nutzen Sie stattdessen ''CNRP FightersGuild4OOO.esp''.'
+      - <<: *useInstead
+        subs: [ 'CNRP FightersGuild4OOO.esp' ]
```
Same situation as above.

```suggestion
      - <<: *useVersion
        subs: [ 'OOO/FCOM' ]
```

sibir-ine reviewed (commented):
@Infernio These could all also use `*useOnlyOneX` messages, but we could always add them later.

Infernio committed:
Refactor CNRP OOO/FCOM messages

I dropped the regex for the DB version entirely, because making them
separate would leave us with an entry for the OOO version that only
consists of plugin name and URL.

Infernio commented:
Refactored them with a new anchor, `useVersionOOO_FCOM`.
